docs/inference.md

```diff
@@ -34,6 +34,15 @@ The inference folder also includes a chat completion example, that adds built-in
 python chat_completion.py --model_name "PATH/TO/MODEL/7B/" --prompt_file chats.json  --quantization --use_auditnlg
 
 ```
+## Loading back FSDP checkpoints
+
+In case you have fine-tuned your model with pure FSDP and saved the checkpoints with "SHARDED_STATE_DICT" as shown [here](../configs/fsdp.py), you can use this converter script to convert the FSDP Sharded checkpoints into HuggingFace checkpoints. This enables you to use the inference script normally as mentioned above.
+**To convert the checkpoint use the following command**:
+```bash
+ python checkpoint_converter_fsdp_hf.py --model_name  PATH/to/FSDP/Checkpoints --save_dir PATH/to/save/checkpoints --model_path PATH/or/HF/model_name
```

Comment:
Will be better to rename the parameters to reflect their usage for conversion. In this case original FSDP checkpoints already exists. Suggest changing to something like below: 

1st param -- fsdp_checkpoint_root PATH/to/FSDP/Checkpoints
2nd param --fsdp_model_name  PATH/to/saved/checkpoints
3rd param --consolidated_model_path Destination location for saving the consolidated model in HF format

Reply:
thanks Geeta, I think maybe we can leave 1&2 FSDP checkpoints as user may be confused how to organize/ recognize these. Changed the third to the suggested option.